src/sentry/seer/similarity/similar_issues.py

```diff
@@ -67,20 +85,47 @@ def get_similarity_data_from_seer(
                 "response_code": response.status,
             },
         )
+        metrics.incr(
+            "seer.similar_issues_request",
+            sample_rate=SIMILARITY_REQUEST_METRIC_SAMPLE_RATE,
+            tags={**metric_tags, "outcome": "error", "error": type(e).__name__},
```

Comment:
let's not include the error type here as a metric tag, this could be too high cardinality.

Reply:
Given that the except block isn't a generic `except Exception`, won't it only ever be one of `AttributeError`, `UnicodeError`, and `JSONDecodeError`? (I used the name attribute just because I didn't want to have to write a separate `except` clause for each of the three error types, only for the purpose of hard-coding the tag value.)

Reply:
sure. we capture the sentry error so i also just think this isn't a very important tag to add

Reply:
We're adding it for every other thing than can go wrong, though, so we need _some_ value there, to distinguish it from getting a bad response status, for example. If we're worried about cardinality, I could make it a more general `DataError` to cover all three cases with one tag, but in point of fact, once we fix Seer to send error JSON rather than the generic Flask error page HTML (causing the JSON decode error), we shouldn't really ever see any of these errors, so I think we're talking more theory than practice here in any case.